geoutils/georaster.py

```diff
@@ -889,8 +904,64 @@ def show(self, band=None, **kwargs):
         else:
             raise ValueError("band must be int or None")
 
+        # If multiple bands (RGB), cbar does not make sense
+        if isinstance(band, collections.abc.Iterable):
+            if len(band) > 1:
+                no_cb = True
+
+        # Create colorbar
+        # Use rcParam default
+        if cmap is None:
+            cmap = plt.get_cmap(plt.rcParams['image.cmap'])
+        elif isinstance(cmap, str):
+            cmap = plt.get_cmap(cmap)
+        elif isinstance(cmap, matplotlib.colors.Colormap):
+            pass
+
+        # Set colorbar min/max values
+        if vmin is None:
```

Comment:
this you could probably leave out as it's just passing arguments, the exceptions should be handled by matplotlib.colors/matplotlib.pyplot

Reply:
Which lines exactly do you mean are not needed? Just the `isinstance(cmap, matplotlib.colors.Colormap)` or the whole condition block?

Reply:
I'm thinking that all the arguments later passed without modification to a matplotlib function should not be checked by us. This is already done by matplotlib

Reply:
The issue is that vmin/vmax are not only used in imshow, but also in ScalarMappable. There, setting them to None results in a colorbar between 0 and 1...

Reply:
Yes that part is perfect, it's the `isinstance(vmin, (int, float, np.integer, np.floating))` that does not seem necessary because will be raised by matplotlib.colors.Normalize, but this is really minor you can leave it

Reply:
I just improved/shortened a bit the condition in commit b45dc66.
Without the try/except, mpl would raise an error that can be difficult to trace.